### PR TITLE
Add instructions

### PR DESCRIPTION
**Pull request by anssiko**
@dontcallmedom @cvan let's use this PR to hash out the instructions, see what should be included. Perhaps this should be on a separate page, but let's get the content ready first.

Preview: https://rawgit.com/w3c/vr-workshop/instructions/schedule.html

### CODE REVIEW

File: schedule.html [hunk 1]
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
...
+        </dt>
+        <dd>
+          Plenary sessions introduce and summarize information relevant to all
+          participants.
+        </dd>
+        <dt>
+          <span class="lightning" id="lightning"></span>
+        </dt>
+        <dd>
+          Discussion topic (30-60 mins) starts off with 3-4 5-minute <a href=
```
Comment (cvan) [inline schedule.html]: would change to `three or four 5-minute`

Comment (cvan) [reply schedule.html]: `mins` and `minute` - should be consistent

File: schedule.html [hunk 2]
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
...
+          speakers nominated in advance. Lightning talks are followed by group
+          discussion and joint summary.
+        </dd>
+        <dt>
+          <span class="breakout" id="breakout"></span>
+        </dt>
+        <dd>
+          Participants break into smaller breakout groups to discuss more
+          specific topics over a period of maximum 60 minutes. The breakout
+          session schedule grid is identified and built collaboratively by the
```
Comment (cvan) [inline schedule.html]: would remove word `grid`

File: schedule.html [hunk 3]
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
...
+          participants in the course of the workshop. Each groups to provide a
```
Comment (cvan) [inline schedule.html]: `groups` to `group is`

File: schedule.html [hunk 4]
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
...
+        <li>Visit <a href="https://slacknow.herokuapp.com/w3c-vr/">auto-inviter
+        service</a> to get an invite, and
+        </li>
+        <li>Join <a href=
+        "https://w3c-vr.slack.com/">https://w3c-vr.slack.com/</a>.
+        </li>
+      </ul>
+      <p>
+        The <a href="https://w3c-vr.slack.com/messages/general/">#general
+        channel</a> is used for plenary and <a href="#lightning">lightning
```
Comment (cvan) [inline schedule.html]: link to `#plenary` as well?

File: schedule.html [hunk 5]
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
...
+        talk</a> discussions. For <a href="#breakout">breakout sessions</a>,
+        groups should create a dedicated Slack channel per breakout.
+      </p>
+      <h4>
+        Meeting notes
+      </h4>
+      <p>
+        Meeting notes will be recorded from all the sessions summarizing the
+        discussions. These notes will be made public. TODO: add instructions
```
Comment (cvan) [inline schedule.html]: perhaps make `TODO` a comment instead?

File: style.css [hunk 6]
```diff
@@ -366,10 +366,62 @@ table.schedule {
   border-spacing: 3px;
 }
 
-table.schedule span {
+table.schedule th span {
   font-size: 67%;
 }
 
+span.lightning, span.plenary, span.breakout, span.panel {
```
Comment (cvan) [inline style.css]: might as well create a common class for these

File: style.css [hunk 7]
```diff
@@ -366,10 +366,62 @@ table.schedule {
...
+span.lightning:before {
+  content: "lightning";
+}
+
+span.plenary {
+  background-color: #b9d3ee;
+}
+
+span.plenary:before {
+  content: "plenary"
```
Comment (cvan) [inline style.css]: `;`

File: schedule.html [hunk 8]
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
...
+          Plenary sessions introduce and summarize information relevant to all
+          participants.
+        </dd>
+        <dt>
+          <span class="lightning" id="lightning"></span>
+        </dt>
+        <dd>
+          Discussion topic (30-60 mins) starts off with 3-4 5-minute <a href=
+          "https://en.wikipedia.org/wiki/Lightning_talk">lightning talks</a> by
+          speakers nominated in advance. Lightning talks are followed by group
```
Comment (cvan) [inline schedule.html:210]: this is mentioned already [above](https://github.com/w3c/vr-workshop/pull/40/files#diff-0953deb6637fd0edbb59e4e408606954R371), but I suppose it's fine to repeat it so people don't miss

File: schedule.html [hunk 9]
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
...
+        <li>
+          <a href=
+          "https://www.w3.org/2002/09/wbs/1/vr-workshop/results">Expressions of
+          interest</a>
+        </li>
+        <li>
+          <a href="papers.html">Position statements</a>
+        </li>
+        <li>
+          <a href="https://surma.link/things/reading-specs/">How to read web
```
Comment (cvan) [inline schedule.html:282]: good link to add 👍

**Review by cvan — CHANGES_REQUESTED** [review]
👍 looks great. just some minor changes.

I especially like what you did with the colour-coding and legend. easy to read.

File: schedule.html [hunk 10]
```diff
@@ -315,6 +355,111 @@ <h4 id="schedule-day-2">
...
+        summaries. The goal of each discussion at the workshop is not to
+        resolve the technical issues of the topic, but to determine its
+        relevance and priority to standardization.
+      </p>
+      <h3 id="format">
+        Format
+      </h3>
+      <dl class="format">
+        <dt>
+          <span class="plenary" id="plenary"></span>
```
Comment (cvan) [inline schedule.html]: seems like it'd be better to just do

``` html
<dt class="format format-plenary" id="plenary">
  Plenary
</dt>
```

File: style.css [hunk 11]
```diff
@@ -366,10 +366,62 @@ table.schedule {
...
 
-table.schedule span {
+table.schedule th span {
   font-size: 67%;
 }
 
+span.lightning, span.plenary, span.breakout, span.panel {
+  display: block;
+  margin: 0 5px 0 0;
+  font-size: 67%;
```
Comment (cvan) [inline style.css]: this looks good in the schedule, but for the legend, the font-size could be bumped up for legibility and some margins could be added I think.

screen shot 2016-10-11 at 1 12 33 pm

screen shot 2016-10-11 at 1 10 31 pm

here are the styles I used:

``` css
dt.format {
  box-shadow: .3em .3em 0 rgba(0,0,0,.1);
  font-size: 1rem;
  font-weight: 600;
  margin: 0 5px 15px 0;
  padding: 8px 10px;
}
```

File: style.css [hunk 12]
```diff
@@ -366,10 +366,62 @@ table.schedule {
...
+  box-shadow: .3em .3em 0 rgba(0,0,0,.1);
+  clear: right;
+}
+
+span.lightning {
+  background-color: #ffe7ba;
+}
+
+span.lightning:before {
+  content: "lightning";
```
Comment (cvan) [inline style.css]: IMO, we should just add normal text content

Comment (cvan) [reply style.css]: (mostly for copy+paste reasons and accessibility)

File: schedule.html (hunk 1 above)
Comment (dontcallmedom) [reply schedule.html]: addressed

File: schedule.html (hunk 2 above)
Comment (dontcallmedom) [reply schedule.html]: addressed

File: schedule.html (hunk 4 above)
Comment (dontcallmedom) [reply schedule.html]: addressed

File: style.css (hunk 11 above)
Comment (dontcallmedom) [reply style.css]: addressed

File: style.css (hunk 12 above)
Comment (dontcallmedom) [reply style.css]: addressed

**Comment by dontcallmedom** [comment]
I've changed the markup for the schedule to allow to add more detailed informations (speakers, related papers, links to slides) in collapsible form

**Comment by anssiko** [comment]
@dontcallmedom @cvan thanks for your review comments and commits!

I think we're now ready to go live with these "instructions v1" (can add more instructions later as needed).

@dontcallmedom WDYT?

**Comment by dontcallmedom** [comment]
@anssiko I'll merge the pre-markup changes to gh-pages and keep that PR open for further iterations

**Comment by anssiko** [comment]
@dontcallmedom Stop press. I think https://github.com/w3c/vr-workshop/pull/40/commits/6fc2dd30b79c314aa9da6b5144cbdd5a3f7dc24d broke the CSS?

**Comment by anssiko** [comment]
See https://rawgit.com/w3c/vr-workshop/instructions/schedule.html

**Comment by dontcallmedom** [comment]
can you be more specific on "broke the CSS" @anssiko ?
(note that as described above, I've merged the clean version to gh-pages)

**Comment by anssiko** [comment]
OK, now I see you did not merge the experimental collapsable items feature yet -- good.

Let's keep iterating that in this PR and integrate the details as well.

**Comment by dontcallmedom** [comment]
oh, I guess the rendering is very different in chrome and in firefox; I now understand better the broken style comment :)

**Comment by dontcallmedom** [comment]
ok, apparently, [`summary` cannot be used as a target for `display:flex` in Chrome](http://stackoverflow.com/questions/30650471/display-flex-doesnt-work-on-summary-tags-in-chrome), will fix

File: schedule.html [hunk 13]
```diff
@@ -92,229 +90,193 @@ <h3 id="silver-sidebar">
...
+        Tools
+      </h3>
+      <h4 id="chat">
+        Chat
+      </h4>
       <p>
-        <b>This schedule may change based on discussions with the <a href=
-        "./#program">program committee</a>.</b>
+        We use Slack during the workshop (offers a lot of IRC-like features).
+        To join the workshop Slash team:
```
Comment (anssiko) [inline schedule.html]: s/Slash/Slack/

**Comment by anssiko** [comment]
Cherry-picking https://github.com/w3c/vr-workshop/commit/defa3619b97a5d077d810fa95c5b1c114852a710 was too painful, so we have a conflict to resolve.

**Comment by cvan** [comment]
do you want me to manually resolve + merge this?

**Comment by anssiko** [comment]
@cvan if you can resolve the conflict introduced by https://github.com/w3c/vr-workshop/commit/defa3619b97a5d077d810fa95c5b1c114852a710 that'd be awesome. I wouldn't yet merge this branch into gh-pages since there are bugs, and @dontcallmedom indicated he's planning to fix them.

File: schedule.html (hunk 13 above)
Comment (cvan) [reply schedule.html]: fixed

**Review by cvan — APPROVED** [review]

**Comment by anssiko** [comment]
To accommodate Philip Rosedale I switched two slots. @cvan It'd be great if you could integrate https://github.com/w3c/vr-workshop/commit/ca7a642c07794db0e2b6c8b50f953a7d7226e65f to this branch.

**Comment by cvan** [comment]
closing in favour of new upcoming PR

**Pull request closed without merging**